manta-js/package/src/BaseWallet.ts

```diff
@@ -23,11 +23,7 @@ export default class BaseWallet implements IBaseWallet {
   getStorageStateFromLocal: GetStorageStateFromLocal;
   walletIsBusy = false;
   isHttpProvider = false;
-
-  static onWasmCalledJsErrorCallback: (
```

Comment:
Wasm is calling the ledger api. If the network fails, it will trigger wasm crash and report an error; this is optimized in wasm, so this error callback is unnecessary